tests/forms_tests/tests/test_forms.py

```diff
@@ -884,7 +893,11 @@ class SongForm(Form):
 <li><label><input checked type="checkbox" name="composers" value="P"> Paul McCartney</label></li>
 </ul>""")
         # Test iterating on individual checkboxes in a template
-        t = Template('{% for checkbox in form.composers %}<div class="mycheckbox">{{ checkbox }}</div>{% endfor %}')
+        e = Engine(debug=True)
+        t = Template(
+            '{% for checkbox in form.composers %}<div class="mycheckbox">{{ checkbox }}</div>{% endfor %}',
+            engine=e,
+        )
```

Comment:
This is missing the same comment that is included in other cases...

It feels like a kludge though. Perhaps we can use a Django version of the template when running the test using the Django engine and a Jinja2 template when using the Jinja2 engine? Perhaps we can overwrite some class attribute in the decorator and access that to check? 😬

Reply:
Yes, it's a bit horrid. My other thought is that these tests are not really Django/Jinja tests. They are tests to ensure you can loop over the choices in `form.composers`. Maybe they could be moved to a separate class?

Reply:
Yes, that sounds sensible. Make it a separate preliminary commit.